src/editor/Neovim/NeovimProcess.re

```diff
@@ -13,10 +13,52 @@ let version = (~neovimPath: string) => {
   ret.stdout;
 };
 
-let start = (~neovimPath: string, ~args: array(string)) => {
-  /* let nvimBinaryPath = Environment.getEnvironmentVariable("ONI2_NEOVIM_PATH"); */
-  ChildProcess.spawn(
-    neovimPath,
-    args,
-  );
+let extractParts = line => {
+  let parts = Str.split(Str.regexp("="), line);
+  switch (parts) {
+  | [name, value] => (name, value)
+  | _ => ("", "")
+  };
+};
+
+let getNeovimPath = paths => {
+  let (_, path) =
+    List.find(
+      ((name, _)) =>
+        switch (name) {
+        | "ONI2_PATH" => true
+        | _ => false
+        },
+      paths,
+    );
+  path;
+};
+
+let getBuildVariablesFromSetup = (~path="/setup.txt", ()) => {
+  let variableList = ref([]);
+
+  let fileInChannel =
+    Pervasives.open_in(Environment.getWorkingDirectory() ++ path);
```

Comment:
Following from the comment earlier about replacing `pwd` with `cur__bin` - can we use `getExecutingDirectory` here and pick up the setup file next to the binary?